CHANGELOG.md

```diff
@@ -6,12 +6,20 @@ All notable changes to Wallaroo will be documented in this file.
 
 ### Fixed
 
+- Gradually back off when attempting to reconnect on data channel
+- Never use more workers than there are partitions
```

Comment:
This should be "There is no longer a problem when using more workers than there are partitions." or something